### PR TITLE
add env vars

### PR DESCRIPTION
**Pull request by decentralgabe**

### CODE REVIEW

File: docker/dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM node:lts-alpine3.14
 
 COPY dist /ion
-COPY json /json
+COPY json /config
```
Comment (thehenrytsai) [inline docker/dockerfile:4]: sanity: should it be:

```suggestion
COPY config /config
```

Comment (decentralgabe) [reply docker/dockerfile:4]: yes

**Review by thehenrytsai — COMMENTED** [review]
Looks great overall with just one minor sanity question.

**Review by thehenrytsai — APPROVED** [review]
Thanks for making configuration more flexible @decentralgabe! 👍 🥇 🎊 🎆 

I'd love to standardize the naming between config params and environment variables, but wouldn't block the PR for it.

**Pull request merged**